<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>843</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>restim</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/restim.qrc">
    <normaloff>:/restim/favicon.png</normaloff>:/restim/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_control">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QFrame" name="left_frame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QStackedWidget" name="stackedWidget_visual">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::NoFrame</enum>
             </property>
             <property name="lineWidth">
              <number>0</number>
             </property>
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="page_threephase">
              <widget class="ThreephaseWidgetAlphaBeta" name="graphicsView_threephase">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>200</width>
                 <height>200</height>
                </rect>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>200</width>
                 <height>200</height>
                </size>
               </property>
              </widget>
             </widget>
             <widget class="QWidget" name="page_fourphase">
              <widget class="FourphaseWidgetStereographic" name="graphicsView_fourphase">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>200</width>
                 <height>200</height>
                </rect>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_volume">
             <property name="title">
              <string>volume</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="VolumeWidget" name="progressBar_volume">
                <property name="value">
                 <number>24</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox_volume">
                <property name="keyboardTracking">
                 <bool>false</bool>
                </property>
                <property name="decimals">
                 <number>2</number>
                </property>
                <property name="maximum">
                 <double>100.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_pattern">
             <property name="title">
              <string>Pattern generator</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QComboBox" name="comboBox_patternSelect">
                <item>
                 <property name="text">
                  <string>Mouse</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Circle</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>A</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>B</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>C</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="doubleSpinBox">
                <property name="keyboardTracking">
                 <bool>false</bool>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>6</number>
          </property>
          <widget class="ThreePhaseSettingsWidget" name="tab_threephase">
           <attribute name="title">
            <string>3-phase</string>
           </attribute>
          </widget>
          <widget class="FourPhaseSettingsWidget" name="tab_fourphase">
           <attribute name="title">
            <string>4-phase</string>
           </attribute>
          </widget>
          <widget class="CarrierSettingsWidget" name="tab_carrier">
           <attribute name="title">
            <string>Carrier settings</string>
           </attribute>
          </widget>
          <widget class="PulseSettingsWidget" name="tab_pulse_settings">
           <attribute name="title">
            <string>Pulse settings</string>
           </attribute>
          </widget>
          <widget class="NeoStimSettingsWidget" name="tab_neostim">
           <attribute name="title">
            <string>NeoStim</string>
           </attribute>
          </widget>
          <widget class="ABTestWidget" name="tab_a_b_testing">
           <attribute name="title">
            <string>A/B testing</string>
           </attribute>
          </widget>
          <widget class="VolumeControlWidget" name="tab_volume">
           <attribute name="title">
            <string>Volume</string>
           </attribute>
          </widget>
          <widget class="VibrationSettingsWidget" name="tab_vibrate">
           <attribute name="title">
            <string>Vibration</string>
           </attribute>
          </widget>
          <widget class="WaveformDetailsWidget" name="tab_details">
           <attribute name="title">
            <string>Details</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="MediaSettingsWidget" name="page_media"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>843</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSetup">
    <property name="title">
     <string>Setup</string>
    </property>
    <addaction name="actionDevice_selection_wizard"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFunscript_conversion"/>
    <addaction name="actionSimfile_conversion"/>
    <addaction name="actionFunscript_decomposition"/>
    <addaction name="separator"/>
    <addaction name="actionFirmware_updater"/>
   </widget>
   <addaction name="menuSetup"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>65</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionControl"/>
   <addaction name="actionMedia"/>
   <addaction name="actionStart"/>
  </widget>
  <action name="actionFunscript_conversion">
   <property name="text">
    <string>Funscript conversion (1d -&gt; 2d)</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionDevice_selection_wizard">
   <property name="text">
    <string>Device selection</string>
   </property>
  </action>
  <action name="actionControl">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/restim.qrc">
     <normaloff>:/restim/sliders_poly.svg</normaloff>:/restim/sliders_poly.svg</iconset>
   </property>
   <property name="text">
    <string>Live Control</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionMedia">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/restim.qrc">
     <normaloff>:/restim/film-1_poly.svg</normaloff>:/restim/film-1_poly.svg</iconset>
   </property>
   <property name="text">
    <string>Sync Media</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionDevice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/restim.qrc">
     <normaloff>:/restim/plug-1_poly.svg</normaloff>:/restim/plug-1_poly.svg</iconset>
   </property>
   <property name="text">
    <string>Device</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="../resources/restim.qrc">
     <normaloff>:/restim/play_poly.svg</normaloff>:/restim/play_poly.svg</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/restim.qrc">
     <normaloff>:/restim/docs-1_poly.svg</normaloff>:/restim/docs-1_poly.svg</iconset>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionSimfile_conversion">
   <property name="text">
    <string>Simfile conversion</string>
   </property>
  </action>
  <action name="actionFunscript_decomposition">
   <property name="text">
    <string>Funscript decomposition</string>
   </property>
  </action>
  <action name="actionFirmware_updater">
   <property name="text">
    <string>Firmware updater</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CarrierSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/carrier_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ThreephaseWidgetAlphaBeta</class>
   <extends>QGraphicsView</extends>
   <header>qt_ui/widgets/threephase_widget.h</header>
  </customwidget>
  <customwidget>
   <class>WaveformDetailsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/waveform_details_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VolumeControlWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/volume_control_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VolumeWidget</class>
   <extends>QProgressBar</extends>
   <header>qt_ui/widgets/volume_widget.h</header>
  </customwidget>
  <customwidget>
   <class>ThreePhaseSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/three_phase_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PulseSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/pulse_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VibrationSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/vibration_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MediaSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/media_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ABTestWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/ab_test_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NeoStimSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/neostim_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FourphaseWidgetStereographic</class>
   <extends>QGraphicsView</extends>
   <header>qt_ui/widgets/fourphase_widget_stereographic.h</header>
  </customwidget>
  <customwidget>
   <class>FourPhaseSettingsWidget</class>
   <extends>QWidget</extends>
   <header>qt_ui/four_phase_settings_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/restim.qrc"/>
 </resources>
 <connections/>
</ui>
